pkg/serviceaccounts/controllers/docker_registry_service.go

```diff
@@ -238,18 +243,18 @@ func getDockerRegistryLocations(lister listers.ServiceLister, location serviceLo
 }
 
 // syncRegistryLocationChange goes through all service account dockercfg secrets and updates them to point at a new docker-registry location
-func (e *DockerRegistryServiceController) syncRegistryLocationChange(key string) error {
+func (e *DockerRegistryServiceController) syncRegistryLocationChange() error {
 	newLocations := e.getDockerRegistryLocations()
 	newDockerRegistryLocations := sets.NewString(newLocations...)
 	existingURLs := e.getRegistryURLs()
-	if existingURLs.Equal(newDockerRegistryLocations) {
-		glog.V(4).Infof("No effective update: %v", newDockerRegistryLocations)
```

Comment:
nit: seems like you may want to keep this

Reply:
fixed, good catch